support/oidc-discovery-provider/README.md

```diff
@@ -7,9 +7,9 @@ exposing a JSON Web Key Set (JWKS) for JSON Web Token (JWT) validation.
 It provides the following endpoints:
 
 | Verb  | Path                                | Description                                                                                                             |
-| ----- | ----------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+|-------|-------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
 | `GET` | `/.well-known/openid-configuration` | Returns the OIDC discovery document                                                                                     |
-| `GET` | `/keys`                             | Returns the JWKS for JWT validation                                                                                     |
+| `GET` | `/keys`                             | Returns the JKS for JWT validation                                                                                      |
```

Comment:
The change to JKS should be reverted to JWKS.

Reply:
Odd. Dunno how that one got in.